LivingCity/traffic/b18TrafficSimulator.cpp

```diff
@@ -175,21 +176,44 @@ void B18TrafficSimulator::simulateInGPU(int numOfPasses, float startTimeH, float
 
     roadGenerationBench.stopAndEndBenchmark();
 
+    Benchmarker edgeOutputting("Edge outputting");
+    edgeOutputting.startMeasuring();
+
+    int index = 0;
+    std::vector<uint> u(graph_->edges_.size());
+    std::vector<uint> v(graph_->edges_.size());
+    for (auto const& x : graph_->edges_) {
+      u[index] = std::get<0>(std::get<0>(x));
+      v[index] = std::get<1>(std::get<0>(x));
+      index++;
+    }
+
+    //save avg_edge_vel vector to file
+    std::string name_u = "./edges_u.txt";
+    std::ofstream output_file_u(name_u);
+    std::ostream_iterator<uint> output_iterator_u(output_file_u, "\n");
+    std::copy(u.begin(), u.end(), output_iterator_u);
+
+    //save avg_edge_vel vector to file
+    std::string name_v = "./edges_v.txt";
```

Comment:
Adding constants in a separate file for these filenames might be good for maintainability of the code.